### ISSUE TITLE
Windows Logon: Incorrect password alert not always read

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4937]
**Reported by Diego on 2015-02-20 20:32**
Hello developers!
to enter the password incorrectly on the logon screen, NVDA does not read the password is incorrect, having to press tab or the command to read the dialogue.

**Comment by josephsl** [comment]
Hi,

Partly addressed in PR #6073 (at least for password entry (not PIN) in Windows 10). Workaround is to press TAB.
Thanks.

**Comment by fisher729** [comment]
@jcsteh, do you think this issue can be closed? I have certainly seen this fixed by issues such as #6010. I haven't seen this happen in a very long time.

**Comment by josephsl** [comment]
Hi, live region change announcement takes care of it, and I did work on specific issues with logonui and touchscreen support. Thanks.

**Comment by jcsteh** [comment]
This works in Windows 10. I'm not sure about the status for Windows 8/8.1. I don't think they use the live region event, but they possibly use the system alert event, which was also implemented in #7282. Any Win 8/8.1 users able to test this?

**Comment by Adriani90** [comment]
I do not have this issue on Windows 8.1. I will close this as works for me. @diego if you still have this issue, please comment on this one and we can reopen it. Thanks.

**Comment by chojiro** [comment]
This still happens in Windows 10 version 1909 for me, as well as previous Windows 10/NVDA releases.

Currently on Windows 10 1909 and NVDA 2019.3.1.

**Comment by feerrenrut** [comment]
@chojiro This issue is old, closed, and doesn't have very specific information. Could you please open a new issue to describe your particular case?

**Comment by chojiro** [comment]
#10881 is open.
